Packs/Core/Scripts/SearchIndicatorInEvents/README.md

```diff
@@ -0,0 +1,32 @@
+Searches for a specific indicator in the tenant's events and logs data and extract the logs which the indicator appears in.
+
+## Script Data
+
+---
+
+| **Name** | **Description** |
+| --- | --- |
+| Script Type | python3 |
+| Tags | Utilities |
+| Cortex XSOAR Version | 6.1.0 |
+
+## Inputs
+
+---
+
+| **Argument Name** | **Description** |
+| --- | --- |
+| indicator | The indicator value \(e.g., IP address, domain, hash\) to search for in the selected dataset logs. |
+| time_frame | The time range to search within, specified in days \(e.g., "7 days" means searching the past 7 days of data\). |
```

Comment:
The search timeframe in days \(e.g., "7 days" means searching the past 7 days of data\).